python/lsst/daf/butler/core/utils.py

```diff
@@ -52,11 +52,15 @@
     Pattern,
     Type,
     TypeVar,
+    TYPE_CHECKING,
     Union,
 )
 
 from lsst.utils import doImport
 
+if TYPE_CHECKING:
+    from ._registry.wildcards import Ellipsis, EllipsisType
```

Comment:
```
from ..registry.wildcards...
```
I think

Reply:
Yup, fixing that now, and mypy complained as well.